torch/nn/utils/rnn.py

```diff
@@ -199,6 +202,27 @@ def pad_packed_sequence(sequence, batch_first=False, padding_value=0):
     return output, Variable(torch.LongTensor(lengths))
 
 
+def _symbolic_pad_packed_sequence(g, input, batch_first=False, padding_value=0.0):
```

Comment:
This seems like a weirdly large amount of boilerplate compared to what we had before. What's going on?

Reply:
So we need two wrappers:

1) A Python function that is called when we execute the PythonOp. This is a nested `def` which wraps the original operation so we can capture the python args `batch_first` and `padding_value`. Here this function is `pad_packed_sequence_trace_wrapper`
2) A symbolic function for the ONNX pass. This is yet another nested def here: `pad_packed_sequence_trace_wrapper`